### PR TITLE
feat: any swift format

### PR DESCRIPTION
**Pull request by gabischima**
*Issue #, if available:* could help #719

*Description of changes:*

- ✨ add new format `ios-swift/any.swift` with:
    - `objectType`: type for the swift object - default is **class**
    - `import`: array or string to import modules - default is **["UIKit"]**
    - `accessControl`: access level for the object - default is **public**
- 🎨 update `ios-swift/class` and `ios-swift/enum` to import `any.swift` template
- 🔥 remove **class.swift.template** and **enum.swift.template**
- ✨ add helper `setSwiftFileProperties` to format file object for swift
- ✅ add test for `setSwiftFileProperties` and snapshot for swift template
- 📝 update docs comment for `ios-swift` formats
- 📝 add `ios-swift/any.swift` to examples / basic / config.json

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by crypt096 — APPROVED** [review]

**Review by crypt096 — APPROVED** [review]

File: __tests__/common/formatHelpers/setSwiftFileProperties.test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+
+const setSwiftFileProperties = require('../../../lib/common/formatHelpers/setSwiftFileProperties');
+
+describe('common', () => {
+  describe('formatHelpers', () => {
+
+    describe('setSwiftFileProperties', () => {
```
Comment (dbanksdesign) [inline __tests__/common/formatHelpers/setSwiftFileProperties.test.js:19]: Thank you for adding tests! 🙌

File: __tests__/formats/swiftFile.test.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions
+ * and limitations under the License.
+ */
+const formats = require('../../lib/common/formats');
+const createDictionary = require('../../lib/utils/createDictionary');
+const createFormatArgs = require('../../lib/utils/createFormatArgs');
+var _ = require('../../lib/utils/es6_');
```
Comment (dbanksdesign) [inline __tests__/formats/swiftFile.test.js:16]: Minor: can we change `var` to `const`?

File: lib/common/formats.js [hunk 3]
```diff
@@ -902,7 +903,7 @@ declare const ${moduleName}: ${JSON.stringify(treeWalker(dictionary.tokens), nul
   },
 
   /**
-   * Creates a Swift implementation file of a class with values
+   * Creates a Swift implementation file of a class with values. It adds default `class` object type, `public` access control and `UIKit` import.
    *
    * @memberof Formats
    * @kind member
```
Comment (dbanksdesign) [inline lib/common/formats.js:909]: Can we add documentation here for the new file options?

File: lib/common/formatHelpers/setSwiftFileProperties.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+/**
+ * Outputs an object with swift format configurations. Sets import, object type and access control.
+ * @memberof module:formatHelpers
+ * @param {Object} file - The file object declare at configuration
+ * @param {String} objectType - Type of the swift object.
+ * @param {String} transformGroup - The transformGroup of the file, so it can be applied proper import
+ * @returns {Object}
+ */
+function setSwiftFileProperties(file, objectType, transformGroup) {
```
Comment (dbanksdesign) [inline lib/common/formatHelpers/setSwiftFileProperties.js:22]: We have been trying to move to using the `options` object on the file to hold configuration like this. If you see in other formats and format helpers, things like `showFileHeader` and `outputReferences` are in an `options` object. Could we do something like that here to be consistent?

File: lib/common/formatHelpers/setSwiftFileProperties.js [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    }
+  } else if (typeof file.import === 'string') {
+    file.import = [file.import];
+  }
+
+  if (typeof file.accessControl === 'undefined') {
+    file.accessControl = 'public ';
+  } else {
+    file.accessControl = `${file.accessControl} `;
+  }
```
Comment (dbanksdesign) [inline lib/common/formatHelpers/setSwiftFileProperties.js:48]: What do you think about removing this (except setting a default accessControl) in favor of placing the trailing space in the swift.any.template file?

**Review by dbanksdesign — COMMENTED** [review]
Overall this looks great! Just a few comments and we can get this merged in. If you are ok with it, when we merge this in we would also like to give credit to @antoniogamizbadger because they started a similar PR as well.

**Comment by chazzmoney** [comment]
@gabischima Would love to get this one merged next week. Any chance you could make the requested changes?

Thanks!

**Comment by gabischima** [comment]
> @gabischima Would love to get this one merged next week. Any chance you could make the requested changes?

yes ! (: i will go into that

File: lib/common/formats.js (hunk 3 above)
Comment (gabischima) [reply lib/common/formats.js:909]: i added doc in the param, do you think i missed something?

File: lib/common/formatHelpers/setSwiftFileProperties.js (hunk 4 above)
Comment (gabischima) [reply lib/common/formatHelpers/setSwiftFileProperties.js]: i updated that (: what do you think?

File: lib/common/formatHelpers/setSwiftFileProperties.js (hunk 5 above)
Comment (gabischima) [reply lib/common/formatHelpers/setSwiftFileProperties.js]: i decided to leave it this way and add the option for empty string `accessControl: ""`. what do you think? in the swift project, when no access is declared, is `internal` by default

File: lib/common/formats.js (hunk 3 above)
Comment (dbanksdesign) [reply lib/common/formats.js:909]: Nope, looks good!

**Review by chazzmoney — APPROVED** [review]

**Pull request merged**